torch/autograd/function.py

```diff
@@ -35,6 +65,27 @@ def mark_dirty(self, *args):
         should be given to this function, to ensure correctness of our checks.
         It doesn't matter whether the function is called before or after
         modification.
+
+        Examples::
+            >>> class Inplace(Function):
+            >>>     @staticmethod
+            >>>     def forward(ctx, x):
+            >>>         x.add_(1)
+            >>>         ctx.mark_dirty(x)
+            >>>         return x
+            >>>
+            >>>     @staticmethod
+            >>>     @once_differentiable  # grad_input is not a function of input
```

Comment:
I don't think this comment makes sense.
The fact that they're independent doesn't make this non-differentiable.

Same for all other custom Functions in this doc